tools/bump_version.sh

```diff
@@ -217,3 +262,60 @@ then
 
     sed -E -i'' -e "s/^(VIProductVersion \").+\"/\1$product\"/g" $NSIS_FILE
 fi
+
+# Wazuh Packages
+if [ -z "$version" ]
+then
+    UPDATE_RELESE_DATE="yes"
+    version=$(cat $VERSION_FILE)
+fi
+
+VERSION=$(sed 's/v//' <<< $version)
+mayor=${VERSION%%.*}
+minor=$(cut -d"." -f1 <<< ${VERSION#*.})
+patch=${VERSION##*.}
+
+if [ -z "$bump_date" ]
+then
+    bump_date=$(LC_ALL=en_US.UTF-8 TZ="UTC" date +"%F")
+    echo "Use default date: $bump_date"
+fi
+
+bump_date=$(LC_ALL=en_US.UTF-8 TZ="UTC" date -d "$bump_date" +"%a, %d %b %Y %H:%M:%S %z")
+
+# SPECS files
+spec_date=$(LC_ALL=en_US.UTF-8 TZ="UTC" date -d "$bump_date" +"%a %b %d %Y")
+for spec_file in $SPEC_FILES; do
+    echo "Bumping version in $spec_file"
```

Comment:
As in the case of _Packages_ there is no version bump, let's change the text to: 
```suggestion
    echo "Updating the release date of $version in $spec_file"
```